packages/plugin-core/src/test/testUtilsV3.ts

```diff
@@ -300,6 +338,7 @@ export function setupBeforeAfter(
     if (opts?.afterHook) {
       await opts.afterHook();
     }
+    sinon.restore();
```

Comment:
Not sure if this will properly restore on Windows; I can investigate this one later.

Reply:
hmm, is there an outstanding issue with sinon on windows?

Reply:
Seems like it...the `sinon.restore()` sandbox style restores doesn't seem to clear the stubs.  I've had to explicitly call restore on the individual stubs objects.